### PR TITLE
Update exception catching

### PR DESCRIPTION
**Pull request by akikanellis**
@dadouf I made a quick fix.

### CODE REVIEW

File: sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java [hunk 1]
```diff
@@ -394,19 +394,23 @@ public void onDestroy() {
...
 
             /*
              * UNBIND
              */
-            context.unbindService(serviceConnection);
+            try {
+                context.unbindService(serviceConnection);
+            } catch (IllegalArgumentException e){
+                FlitchioLog.w("Warning: it seems that you tried to call onDestroy without" +
+                        " having a bind to Flitchio Manager");
```
Comment (dadouf) [inline sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java]: Oups, spelling: bind -> binding.

File: sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java [hunk 2]
```diff
@@ -394,19 +394,23 @@ public void onDestroy() {
...
                 FlitchioLog.w("Warning: binding to Flitchio Manager not yet effective.");
             }
 
             /*
              * UNBIND
              */
-            context.unbindService(serviceConnection);
+            try {
+                context.unbindService(serviceConnection);
+            } catch (IllegalArgumentException e){
```
Comment (dadouf) [inline sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java]: I think the formatter should put a space between ) and { there, isn't it?

Comment (akikanellis) [inline sdk/src/main/java/com/supenta/flitchio/sdk/FlitchioController.java]: My formatter in my laptop is indeed a bit fucked, I need to import the settings from the office computer.

**Comment by dadouf** [comment]
Approval from me :+1:

**Pull request merged**